src/main/resources/bridge-sdk.properties

```diff
@@ -1,5 +1,5 @@
 log.level = warn
-sdk.version = 1
+sdk.version = 2
```

Comment:
I wanted to create a version that matched version 1 of the server, the first version that cannot be changed without assessing backwards compatibility. In truth, nothing looks at this version right now. But in the logs, version 2 of the SDK is the SDK written against the v1 release of the server.